.github/workflows/pr_tests.yml

```diff
@@ -130,7 +130,7 @@ jobs:
       if: ${{ matrix.config.framework == 'pytorch_pipelines' }}
       run: |
         python -m venv /opt/venv && export PATH="/opt/venv/bin:$PATH"
-        python -m pytest -n 2 --max-worker-restart=0 --dist=loadfile \
+        python -m pytest -n 4 --max-worker-restart=0 --dist=loadfile \
```

Comment:
If the runner has 32 cores available for pipelines why not bump this up further? To 8?

Reply:
Done in 9afd3cca. 
🤞
Let's see